pyasdf/utils.py

```diff
@@ -1330,3 +1330,14 @@ def to_list_of_resource_identifiers(obj, name, obj_type):
             msg = "Invalid type for %s." % name
             raise TypeError(msg)
     return [obj]
+
+
+def replace_None(param):
```

Comment:
Can you please rename this to `replace_none()` to follow the naming conventions.